src/olympia/core/languages.py

```diff
@@ -214,6 +214,7 @@
         'native': '\u1c65\u1c5f\u1c71\u1c5b\u1c5f\u1c72\u1c64',
     },
     'si': {'english': 'Sinhala', 'native': '\u0dc3\u0dd2\u0d82\u0dc4\u0dbd'},
+    'skr': {'english': 'Saraiki', 'native': '\u0633\u0631\u0627\u0626\u06cc\u06a9\u06cc'},
```

Comment:
Linting doesn't like that line, try running `ruff format` on the file or apply

```suggestion
    'skr': {
        'english': 'Saraiki',
        'native': '\u0633\u0631\u0627\u0626\u06cc\u06a9\u06cc',
    },
```